src/frontend/src/catalog/system_catalog/rw_catalog/rw_parallel_units.rs

```diff
@@ -19,26 +19,24 @@ use crate::catalog::system_catalog::SysCatalogReaderImpl;
 use crate::error::Result;
 
 #[derive(Fields)]
-struct RwParallelUnit {
+struct RwWorkerSlot {
+    #[primary_key]
+    slot_id: i32,
     #[primary_key]
-    id: i32,
     worker_id: i32,
 }
 
 #[system_catalog(table, "rw_catalog.rw_parallel_units")]
-fn read_rw_parallel_units(reader: &SysCatalogReaderImpl) -> Result<Vec<RwParallelUnit>> {
+fn read_rw_parallel_units(reader: &SysCatalogReaderImpl) -> Result<Vec<RwWorkerSlot>> {
```

Comment:
nit: it might worth adding comments here to explain why `rw_parallel_units` returns `RwWorkerSlot`